valhalla/config.h

```diff
@@ -12,6 +12,9 @@
 
 // clang-format off
 #define VALHALLA_VERSION VALHALLA_STRINGIZE(VALHALLA_VERSION_MAJOR) "." VALHALLA_STRINGIZE(VALHALLA_VERSION_MINOR) "." VALHALLA_STRINGIZE(VALHALLA_VERSION_PATCH)
+#ifdef VALHALLA_VERSION_MODIFIER
```

Comment:
hm, isn't this always defined? I'm not sure how to read `$<$<BOOL:${VALHALLA_VERSION_MODIFIER}>:VALHALLA_VERSION_MODIFIER=${VALHALLA_VERSION_MODIFIER}>` in the CMakeLists.txt but at least for Docker & the python ones it's always defined.

would probably be better to only add this compile option if ``$VERSION_MODIFIER == ""`? not the biggest issue, but now it'll probably show like `3.5.1-` if there's no git hash?